### ISSUE TITLE
Make it clearer that the real upload of the data only goes through after the local processing of the data

### ISSUE THREAD

**Issue by ri72miieop** [TheExGenesis/community-archive#118]
we could make it more explicit in the "Upload to the Community Archive!" page that the data only leaves the user computer after we process it locally and extract the info from the <explicit list of files> we use. I feel like for a certain type of user that is more privacy-aware this would be helpful

**Comment by TheExGenesis** [comment]
yeah I used to have it in the front page of the website but that felt like too much. I made [this page](https://github.com/TheExGenesis/community-archive/blob/main/docs/archive_data.md) for more transparency on what data gets used and why, could add a few lines there. 

What else do you think we should do?

**Comment by ri72miieop** [comment]
Yeah that page is perfect. Maybe we could add emojis to the 3 items on "Useful links:" 1st ℹ️ 2nd 🔒 (or other that the user might associate with privacy) and 3rd maybe 🤝 to make them more engageable/parsable and add that link on the 2nd item. With the 3 links people should have all the info they need.

**Comment by karol-majewski** [comment]
I think it could be useful to inform people what data will be used right on this screen because "You can download any individual user's data (includes all tweets, followers, following, etc) as one big JSON file" sounds like a repellent to anyone privacy-oriented

![Image](https://github.com/user-attachments/assets/76a96b6b-4ac1-4fb8-9689-ce7714554d46)

These can be considered sensitive, so it might help to be upfront about Community Archive _not_ caring about these:

- Direct messages
- DIrect message media
- Conversations with Grok
- Deleted tweets
- Deleted tweets media
- Lists
- Anything containing phone numbers and e-mails
- Auth tokens, key registry
- Likes (which are optional afaik)
- Blocked and muted accounts
- ageinfo, date of birth
- ad impressions
- ad engagement
